Doc/library/dis.rst

```diff
@@ -723,6 +723,47 @@ All of the following opcodes use their arguments.
    are put onto the stack right-to-left.
 
 
+.. opcode:: BUILD_TUPLE_UNPACK (count)
```

Comment:
Following opcodes create the collection by concatenating/merging several values from the stack. I think it would be better to place their descriptions after `BUILD_STRING` (which is similar to `BUILD_*_UNPACK` opcodes but for strings).